shared/js/project.js

```diff
@@ -274,7 +274,14 @@ var projects = module.exports = {
     });
   },
   serverSideRemix: function() {
-    location.href = projects.getPathName('remix');
+    if (current && !current.name &&
```

Comment:
Do we also want to set "Untitled Project" as a default if `current && !current.name` but `!projects.appToProjectUrl() === '/projects/algebra_game'`?